blockchain_exploration/exploration.py

```diff
@@ -222,5 +237,9 @@ def get_network_by_token_type(token_type: str) -> str:
         return Network.TEZOS
     elif token_type == TokenType.SLP:
         return Network.BITCOIN_CASH
+    elif token_type == TokenType.TON:
+        return Network.TON
+    elif token_type == TokenType.SCOR:
+        return Network.TON
     else:
         raise NotImplementedError(f"No network known for {token_type}")
```

Comment:
The missing EOL suggests that you've not run the pre-commit hooks....

Reply:
will update